src/main/java/com/google/devtools/build/lib/query2/cquery/ConfiguredTargetQueryEnvironment.java

```diff
@@ -219,7 +220,9 @@ private static BuildConfiguration mergeEqualBuildConfiguration(
         cqueryOptions.aspectDeps.createResolver(packageManager, eventHandler);
     return ImmutableList.of(
         new LabelAndConfigurationOutputFormatterCallback(
-            eventHandler, cqueryOptions, out, skyframeExecutor, accessor),
+            eventHandler, cqueryOptions, out, skyframeExecutor, accessor, confiredTargetAccessor, true),
+        new LabelAndConfigurationOutputFormatterCallback(
+            eventHandler, cqueryOptions, out, skyframeExecutor, accessor, confiredTargetAccessor, false),
```

Comment:
Following my longer comment: you don't need `configuredTargetAccessor` here or in the above line.